vectortile/src/main/scala/geotrellis/vectortile/VectorTile.scala

```diff
@@ -45,9 +45,9 @@ import geotrellis.util.annotations.experimental
   * @constructor This is not meant to be called directly - see this class's
   * companion object for the available helper methods.
   */
-@experimental case class VectorTile(layers: Map[String, Layer], tileExtent: Extent) {
+@experimental case class VectorTile(layers: Map[String, Layer], tileExtent: Extent, forceWindClockwise: Boolean = true) {
```

Comment:
The more I look at this, the more I prefer `forcePolygonWinding`.  First, clockwise winding isn't _exactly_ right, given the interior holes are wound anti-clockwise.  Second, it only applies to polygons, not to closed linestrings.  Last (and most nitpicky), it doesn't read as proper English (`forceClockwiseWinding` would have been better).

This is very pedantic, but I feel like we get tied down to these early nomenclature choices and then end up having to live with them.  See raster/spark reproject options for an example of this gone to the worst extreme.